user/api.go

```diff
@@ -181,7 +180,7 @@ func (a *Api) GetUsers(res http.ResponseWriter, req *http.Request) {
 		a.sendError(res, http.StatusInternalServerError, STATUS_ERR_FINDING_USR, err.Error())
 
 	} else {
-		a.logMetric("getusers", sessionToken, map[string]string{"server": strconv.FormatBool(tokenData.IsServer)})
+		a.logMetric("getusers", req.Header.Get(TP_SESSION_TOKEN), map[string]string{"server": strconv.FormatBool(tokenData.IsServer)})
```

Comment:
So, what happens with all of these `req.Header.Get(TP_SESSION_TOKEN)` metrics if it uses an access token (no the Tidepool session token). I'd suggest you add an accessor of the `tokenData` struct to get the correct token for the `logMetric` call.